app/src/main/java/org/simple/clinic/widgets/BottomSheetAnimations.kt

```diff
@@ -0,0 +1,56 @@
+package org.simple.clinic.widgets
+
+import android.view.View
+import androidx.interpolator.view.animation.FastOutSlowInInterpolator
+
+/*
+ * These methods were extracted out of BottomSheetActivity because they are also used in
+ * LinkIdWithPatientView, which is not a BottomSheetActivity, but needs to behave like one.
+ **/
+
+fun showBottomSheet(
```

Comment:
We should rename this method to specify that we are adding animations using this method.
Wdyt about `showBottomSheetWithAnimation`?